charts/operator/deployment.go

```diff
@@ -366,8 +357,6 @@ func operatorArguments(dot *helmette.Dot) []string {
 		"--health-probe-bind-address=:8081",
 		"--metrics-bind-address=127.0.0.1:8080",
 		"--leader-elect",
-		fmt.Sprintf("--configurator-tag=%s", configuratorTag(dot)),
```

Comment:
Should the corresponding values be removed or marked as deprecated as they're now a no-op?

Reply:
I can mark it as deprecated. My intent was to remove argument form deployment arguments to not confuse end user.

Reply:
After more thought both option should stay as is. Any user could potentially want to overwrite the "configurator" container registry and it's tag. It shouldn't be encourage, but some K8S cluster can be cut from the internet. Internal container registry would be then used and user is free to specify helm chart `additionalCmdFlags` to pass `configurator-tag` and `configurator-base-image` values

Reply:
> but some K8S cluster can be cut from the internet

Great call out!

Reply:
Does that mean you'll be reverting the change this comment thread is on?